apps/aecore/lib/aecore/naming/name.ex

```diff
@@ -23,7 +23,7 @@ defmodule Aecore.Naming.Name do
   @typedoc "Structure of the Name Transaction type"
   @type t :: %Name{
           hash: binary(),
-          owner: Keys.pubkey(),
+          owner: Identifier.t(),
```

Comment:
This will break serialization of the name structure:
https://github.com/aeternity/elixir-node/blob/c76f802d029419e90ab86d841948c1bc3d4349fa/apps/aecore/lib/aecore/naming/name.ex#L94